container.go

```diff
@@ -304,6 +305,7 @@ func (c *Container) initialCleanup(ctx context.Context) {
 			if err = c.cli.ContainerRemove(ctx, cont.ID, types.ContainerRemoveOptions{
 				Force:         true,
 				RemoveVolumes: true,
+				RemoveLinks:   true,
```

Comment:
what does this do?

Reply:
Links are a legacy networking feature that allowed containers to communicate with each other. Setting `RemoveLinks`  to `true` here removes those legacy links on the network, if they existed.